### ISSUE TITLE
Data Contract from multiple XSD files

### ISSUE THREAD

**Issue by GrapeJamMan** [WSCF/WSCF#4]
From the feature list:

- A Generate Data Contract Code feature that supports the selection of multiple XSD/WSDL source files.

The link after it is a dead link. Selecting multiple XSD files won't bring up with WSCF.blue menu. When a single XSD file is selected, WSCF.blue | Generate Data Contract Code...  does show up, and the dialog suggests multiple XSD files can be selected: the label reads "XSD files(s)". But I can't alter the filenames in that box either.

Is there a UI trick to getting this to work as descrbied?

**Comment by Turenkarn** [comment]
Yup, also huge issue for me, actually makes the tool quite useless...
The working link is here: [link](http://alexmg.com/posts/data-contract-generation-is-now-available-in-wscfblue)
But since then they have messed around with the context menu and there is no way to pass multiple .xsd files to the tool anymore... The WSCF.Blue is gone from the context menu when you select multiple files, and the XSD File(s) field on the interface is also not editable.

It seems to have worked with multiple files, maybe it still would, but the UI won't let you do it anymore...

**Comment by GreenUtil** [comment]
Indeed, the only possible file type for the WSCF extension in Visual Studio is **.wsdl**. 
**.xsd** is not supported (anymore ?).
I am wondering how the **WSCF.Blue** tool compares to **Svcutil.exe** and **xsd.exe**  

![image](https://user-images.githubusercontent.com/2122311/46273959-ea634100-c557-11e8-83d1-d69e173cec62.png)

**Comment by GreenUtil** [comment]
When I add an  .xsd file to the project and right-click on it, then a **Generate Data Contract Code** option comes available....so it works !


![image](https://user-images.githubusercontent.com/2122311/46281315-9f085d00-c56e-11e8-8b39-ba4e9324460b.png)
